pmlc/conversion/tile_to_pxa/tile_to_pxa.cc

```diff
@@ -414,6 +415,16 @@ static Value createInit(OpBuilder &builder, Location loc, Type type,
       auto value = convertFloatUsingType(llvm::APFloat(1.0), floatType);
       return builder.create<mlir::ConstantFloatOp>(loc, value, floatType);
     }
+    case AggregationKind::min: {
+      auto value = convertFloatUsingType(
+          llvm::APFloat(std::numeric_limits<float>::max()), floatType);
```

Comment:
Also, we probably want to use max of `double` rather than `float`. Actually, let's see if there's an LLVM way to create a `APFloat` max and min.

Reply:
There is an `APFloat::getInf`, but it requires you to specify the float semantics; I went with just constructing from a double in PR #1064 because that seemed clearer, but perhaps I'm wrong.